ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -142,4 +142,6 @@
     private CCDLaList assignedTo;
     private CCDContactPartyType contactChangeParty;
     private CCDContactChangeContent contactChangeContent;
+    private CCDProceedOnPaperRequestType proceedOnPaperReason;
```

Comment:
To be consistent with the other property names,  name this property 'proceedOnPaperRequestType'